lib/kernel.pm

```diff
@@ -28,11 +29,19 @@ sub remove_kernel_packages {
     my @packages;
 
     if (check_var('SLE_PRODUCT', 'slert')) {
-        @packages = qw(kernel-rt kernel-rt-devel kernel-source-rt);
+        # workaround for bsc1227773
+        if (is_transactional) {
+            @packages = qw(kernel-rt);
+        } else {
+            @packages = qw(kernel-rt kernel-rt-devel kernel-source-rt);
+        }
```

Comment:
This would be cleaner:

    @packages = qw(kernel-rt);
    push @packages, qw(kernel-rt-devel kernel-source-rt) unless is_transactional;